src/Mailer/Transport/SmtpTransport.php

```diff
@@ -165,6 +172,16 @@ public function getLastResponse(): array
         return $this->_lastResponse;
     }
 
+    /**
+     * Returns the authentication type detected and used to connect to the SMTP server.
+     * If no authentication was detected, null is returned.
+     * @return string|null
+     */
+    public function getAuthType(): ?string
```

Comment:
This method doesn't seem to have any use outside of testing.

Reply:
Thanks for the feedback! Method was moved to the test class.

There are two builds failing that don't seem related to this PR. Please advise.

Reply:
You can ignore the unrelated failures.